payments/models.py

```diff
@@ -262,6 +263,33 @@ def update_expired(self) -> int:
 
         return too_old_waiting_orders.count()
 
+    def get_queryset(self):
```

Comment:
If paid_at/rejected_at/cancelled_at is not needed, is there any significant performance overhead added to the queries? I don't know if Django is able to optimize the query, and if it matters here at all. If there is, should there be an additional manager instead of overriding the default manager?

I think the changes can be merged as is, optimize later if there's need

Reply:
It doesn't add that much overhead by executing it like this, but if several order instances need it, it gets quite heavy. Regarding the manager, it's better to keep it like that. It doesn't really add any complications or anything and does not modify the original manager. On top of that, Grapehene internally relies on the `.objects` manager to do the magic, so if we were to put this annotations on a different manager, we would have to manually add all the resolvers.